### ISSUE TITLE
latest update brakes night theme 

### ISSUE THREAD

**Issue by computersalat** [sreichholf/dreamDroid#89]
since latest update (> 1.1-386) night theme is not working... 
with 1.1-386 and previous. it worked just fine. 
running RR on OP3

**Comment by FloEdelmann** [comment]
I guess it was my change that broke instant theme change. After a restart, the new theme still applies.

**Comment by sreichholf** [comment]
Yes. @FloEdelmann do you want to go for it or shall i fix it? (I wouldn't mind doing it, but i don't wanna take it from you).

**Comment by FloEdelmann** [comment]
@sreichholf I can't look into it until Tuesday. If you want to fix it earlier, I have nothing against that :)

**Comment by sreichholf** [comment]
fixed in db5b292e13b99914b67af511c4d80357893483dc (and 95332b2532361915a91c996d1624dc61dd78f7f1)

**Comment by computersalat** [comment]
almost ... but not 100% ...
I am using **Night Theme**. When installing now I have a mix of **Day and Night** theme ... never had this before ...
But now it can be fixed with switching to **Day Theme** and back to **Nigh Theme** ...
... probably you should sort it out ...

**Comment by sreichholf** [comment]
Couldn't reproduce it but I've removed a small portion of code that wasn't required and may have caused some "chaos".

**Comment by computersalat** [comment]
I am coming from ```1.1-386``` ...
noticed this with updating to ```1.1-396```
Now, with latest update from ```1.1-396``` to ```1.1-397``` it is even worse now, even switching between themes does not fix.
Have a look at the screenshots:
mixed **day/night** 
![Uploading Screenshot_20170227-165741.png…]()
theme after update
![Uploading Screenshot_20170227-165741.png…]()
clean **day theme** after reverting update to ```1.1-386```

**Comment by computersalat** [comment]
`Aussitzen` ist auch eine Variante des Bugfixing ;)

**Comment by sreichholf** [comment]
Bin ich denn jetzt verpflichtet ständig an der App zu arbeiten?
Ich mach das wenn ich Lust dazu hab und wenn nicht, dann eben nicht. Gerade in den Sommermonaten passiert i.d.R. sehr wenig bis garnichts. Soviel Geduld wird man mitbringen müssen.

**Comment by computersalat** [comment]
Wenn Du meinen `Sarkasmus` falsch verstanden hast, dann tut es mir Leid.
Ich erwarte in keinster Weise, daß Du Dich verpflichtet fühlst ständig an der App zu arbeiten. Aber dein Kommentar bzgl `Sommermonate` finde ich unpassend, denn März, April und Mai sind IMHO keine Sommermonate.
Eigentlich wollte ich Dir nur weiteres, intensives nachforschcen erparen, da der Fehler auch nicht mehr aufgetreten ist, deswegen habe ich den auch geschlossen.
Nun denn, wenn es Dein Anspruch ist diesem Verhalten auf den Grund zu gehen, dann lass Dich bitte nicht beirren.
schönen Sommertag noch ;)

**Comment by sreichholf** [comment]
Man muss halt dazu sagen dass die Funktionalität von Google bereitgestellt wird. Leider gabs da in der Vergangenheit immer wieder merkwürdige Probleme beim Umschalten di eman irgendwie "umschiffen" musste...

**Comment by sreichholf** [comment]
Closing: likely fixed on the 2.0 rewrite (main).
